lua/luasnip/util/path.lua

```diff
@@ -2,20 +2,13 @@ local Path = {}
 
 local uv = vim.loop
 
-local sep = (function()
-	if jit then
-		local os = jit.os:lower()
-		if vim.tbl_contains({ "linux", "osx", "bsd" }, os) then
-			return "/"
-		else
-			return "\\"
-		end
-	end
-	return package.config:sub(1, 1)
-end)()
+-- In windows, unix style of paths is also valid
+function Path.unixfy(path)
+	return path:gsub("\\", "/")
```

Comment:
A unix path could contain `\` naturally :/

Reply:
Didn't though about that (neither neovim devs :smile: )

Reply:
:grimacing::grimacing:

Reply:
I'll use the `cappital_letter:` pattern to detect if it's a windows path.

Reply:
Are we sure relative paths won't be passed to unixify?

Reply:
We are the only users of it,  it's used in `join` and we only call join starting from absolute paths, the other usage is `expand` but that one resolves to an absolute path before calling it.

Reply:
Mhmm, okay it would work for now, but just thinking of the unexpected errors and debugging _if_ a relative path somehow makes its' way there.. :grimacing:
(we could rename all functions to reflect that they're supposed to handle only relative paths, but that's hardly a good solution)

I think the proper way to tackle this issue is to improve the normalization of paths, so always changing `\` to `/`, or, if they're relative, remove leading `./` (for this problem, for example, it would be enough to remove the `./` from paths in `package.json`).

But, I'd wait until we understand the full behaviour of `fs_realpath` on windows, then we can make up for corner-cases not covered by it.